assets/course-theme/focus-mode.ts

```diff
@@ -23,7 +23,7 @@ const restoreFocusModeState = () => {
 	const savedState = window.sessionStorage.getItem( FOCUS_MODE_CLASS );
 	if ( ! savedState ) return;
 	try {
-		const wasActive = JSON.parse( savedState );
+		const wasActive: unknown = JSON.parse( savedState );
```

Comment:
is it an object boolean? 🤔

Reply:
Apparently, yeah! haha 

I marked as unknown because we have a `typeof` check on the next line, so..yeah. It might or might not be defined. :(